wpeverest/user-registration
# Enhance/selective countries

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds a feature of `Selective Countries` to country fields, which allows admin to restrict the country options for users to select from.



### How to test the changes in this Pull Request:

1. Add a `Country` field
2. Go to `Field Options` tab
3. In the `Advance Settings` section, there's a `Selected Countries` option.
4. Select a few countries.
5. Save the form and go to the frontend.
6. Now, the country field should have only the selected countries as options.
7. Try to register and see if everything is working properly.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Add `Selective Countries` feature to country fields.
